src/peft/tuners/tuners_utils.py

```diff
@@ -494,6 +500,31 @@ def _unloading_checks(self, adapter_names: Optional[list[str]]):
         if is_modules_to_save_available and len(adapters_to_consider) > 1:
             raise ValueError("Cannot unload multiple adapters that specify `modules_to_save`.")
 
+    @staticmethod
+    def get_model_config(model: nn.Module, default: dict | None = {"model_type": "custom"}) -> dict:
```

Comment:
Let's avoid having a dictionary as a default argument. Generally in Python, we should not use mutable default arguments. I think the best alternative is to simply remove `default` as an argument entirely, just hard-code `{"model_type": "custom"}`.

I guess we could also create a constant `{"model_type": "custom"}` inside of `utils/constants.py` and use it consistently in the code base, up to you if you want to go that far.

Reply:
I did this as in one case [here](https://github.com/huggingface/peft/blob/900f96c40ddebae9d76bed374c8baed60e8b34e9/src/peft/peft_model.py#L1234), `default` is `None`. 

I'll try handling that `None` separately.

Reply:
I just did [this](https://github.com/huggingface/peft/pull/2025/files#diff-95787fcc5356568929e383449ade2c1bac5da43deccab17318652b62ed171ae7R1237) as it's a bit unclear if in this case the `model_config` needs to default to `None` or if it can be the DUMMY one, let me know!